data.json

```diff
@@ -209,7 +209,7 @@
   {
     "name": "SimbaChain",
     "difficulty": "easy",
-    "description": "Register and get a free Blockchain Shirt! Pick your side, 'eth_listtransactions=_yep' vs 'eth_listtransactions=_nope'.",
+    "description": "Register and get a free Blockchain Shirt! Pick your side, <code>'eth_listtransactions = _yep'</code> vs <code>'eth_listtransactions=_nope'</code>.",
```

Comment:
You want to delete the quotes and add back space between 'transactions = _nope'

Reply:
what kind of styles do you think, i was thinking font-size and maybe color